backend/.env.bak

```
@@ -0,0 +1,164 @@
+# SECURITY WARNING: Change these default values in production!
+# Generate secure random keys using: openssl rand -hex 32 (for key) and openssl rand -hex 16 (for IV)
+SHARE_TOKEN_AES_KEY=12345678901234567890123456789012
+SHARE_TOKEN_AES_IV=1234567890123456
+
+# YAML initialization configuration
+# Directory containing YAML files for initial data (default: /app/init_data)
+INIT_DATA_DIR=/app/init_data
+# Enable/disable YAML initialization on startup (default: True)
+INIT_DATA_ENABLED=True
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)